lib/MongoDBKeyDescriptionStorage.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {KeyDescriptionStorage} = require('web-kms-switch');
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+const {promisify} = require('util');
+const {util: {BedrockError}} = bedrock;
+
+// load config defaults
+require('./config');
+
+bedrock.events.on('bedrock-mongodb.ready', async () => {
+  await promisify(database.openCollections)(['kms']);
+
+  await promisify(database.createIndexes)([{
+    // cover queries by ID
+    collection: 'kms',
+    fields: {id: 1},
+    options: {unique: true, background: false}
+  }, {
+    // cover queries by controller
+    collection: 'kms',
+    fields: {controller: 1},
```

Comment:
Do we need this index and field, there is no query API that uses this at this time.